.github/workflows/py-tests.yml

```diff
@@ -23,6 +23,9 @@ on:
       - ingestion/**
       - catalog-rest-service/**
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
We need to use fallback value here as there are multiple events that this CI is listening to.
Ref - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value